packages/shadydom/src/patches/Node.js

```diff
@@ -375,6 +375,12 @@ export const NodePatches = utils.getOwnPropertyDescriptors({
     } else if (node.ownerDocument !== this.ownerDocument) {
       this.ownerDocument.adoptNode(node);
     }
+    if (!utils.disableShadowParts) {
```

Comment:
See previous comment about performance concerns and consider instead a tighter integration at https://github.com/webcomponents/polyfills/blob/master/packages/shadydom/src/patches/Node.js#L311-L333.

Reply:
Acknowledged. Added TODO and filed https://github.com/webcomponents/polyfills/issues/345 for upcoming task.